src/libImaging/ImPlatform.h

```diff
@@ -37,15 +37,37 @@
 #undef WIN32
 #endif
 
-#else
+#else /* WIN */
```

Comment:
The if block is for WIN, the else block is for not WIN?

```suggestion
#else /* not WIN */
```